app/models/user.rb

```diff
@@ -13,7 +21,7 @@ class User < ApplicationRecord
 
   enum gender: {female: 0, male: 1, other: 2}
 
-  scope :latest_first, -> {order(created_at: :desc)}
+  scope :latest_first, -> {order(created_at: :asc)}
```

Comment:
```suggestion
  scope :latest_first, -> {order(created_at: :desc)}
```
lastest thì phải desc chứ, chỗ này khi sửa logic, e phải check lại tên hàm nữa nhé